tests/Microsoft.Identity.Test.Integration.netcore/HeadlessTests/UsernamePasswordIntegrationTests.NetFwk.cs

```diff
@@ -82,20 +82,29 @@ public async Task ARLINGTON_ROPC_AAD_CCA_Async()
 
         [RunOn(TargetFrameworks.NetCore)]
         [TestCategory(TestCategories.Arlington)]
+#if IGNORE_FEDERATED
+    [Ignore]
```

Comment:
Nit: indentation. Here and for all other Ignore tags in this class

Reply:
fixed it